cupy_backends/cupy_rand.h

```diff
@@ -3,7 +3,8 @@
 
 #if CUPY_USE_HIP
 
-#include "hip/cupy_hiprand.h"
+//#include "hip/cupy_hiprand.h"
+#include "hip/cupy_hiprand_new.h"
```

Comment:
Please update file path to "hip/cupy_hiprand.h" as this will now be the official supported header.

Reply:
Done.